### PR TITLE
[#2339] Fix `VaDataTable` css variables (for `sticky` and `scrolled` table)

### LINKED ISSUE #2339: Date table impossible to change background color css variable
image

It set on element, not on classs or :root.

### PR DESCRIPTION
**Pull request by RVitaly1978**
## Description
close #2339 

changes:
- [x] add css variable for sticky background-color
- [x] update docs example

chrome-capture-2022-8-12 (1)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

**Review by aluarius — APPROVED** [review]

File: packages/ui/src/components/va-data-table/VaDataTable.vue [hunk 1]
```diff
@@ -403,7 +400,8 @@ export default defineComponent({
         { 'va-data-table--scroll': !!props.height },
         attrs.class,
       ],
-      style: [stickyCSSVariables.value, attrs.style],
+      style: [attrs.style],
+      // style: [stickyCSSVariables.value, attrs.style],
```
Comment (rustem-nasyrov) [inline packages/ui/src/components/va-data-table/VaDataTable.vue:404]: Was it on purpose or you forgot to remove it?

File: packages/docs/src/page-configs/ui-elements/data-table/examples/Sticky.vue [hunk 2]
```diff
@@ -16,8 +16,8 @@
     <va-data-table
       :items="items"
       :style="{
-        '--va-data-table-scroll-table-height': '300px',
-        '--va-data-table-scroll-table-color': 'orange',
+        '--va-data-table-scrolled-table-height': '300px',
```
Comment (m0ksem) [inline packages/docs/src/page-configs/ui-elements/data-table/examples/Sticky.vue:19]: How about `--va-data-table-height`?

File: packages/docs/src/page-configs/ui-elements/data-table/examples/Sticky.vue [hunk 3]
```diff
@@ -16,8 +16,8 @@
...
+        '--va-data-table-sticky-background-color': 'orange',
```
Comment (m0ksem) [inline packages/docs/src/page-configs/ui-elements/data-table/examples/Sticky.vue:20]:
```suggestion
        '--va-data-table-header-background': 'orange',
```

```css
--va-data-table-header-background: linear-gradient(0deg, #f0f, #fff);
```

**Review by m0ksem — APPROVED** [review]

File: packages/docs/src/page-configs/ui-elements/data-table/examples/Sticky.vue (hunk 2 above)
Comment (RVitaly1978) [reply packages/docs/src/page-configs/ui-elements/data-table/examples/Sticky.vue:19]: done

File: packages/docs/src/page-configs/ui-elements/data-table/examples/Sticky.vue (hunk 3 above)
Comment (RVitaly1978) [reply packages/docs/src/page-configs/ui-elements/data-table/examples/Sticky.vue:20]: done

**Pull request merged**